src/app/validate-iati/validate-result/validate-result.component.ts

```diff
@@ -71,12 +77,12 @@ export class ValidateResultComponent implements OnDestroy {
     if (!this.iatiDatasetDatas) {
       return false;
     } else {
-      return this.jsonUpdated(this.iatiDatasetDatas);
+      return this.iatiDatasetDatas.every(iatiDatasetData => this.jsonUpdated(iatiDatasetData));
```

Comment:
it looks like this change stops the polling for updates in the list of files, without this change the polling works